extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/local_inference/text_summarization.py

```diff
@@ -128,13 +128,18 @@ def construct_stream_output(
             "metadata": {},
         }
     )
+
     accumulated_message = ""
     for new_text in streamer:
         if isinstance(new_text, str):
+            # For some reason these symbols aren't filtered out by the streamer
+            new_text = new_text.replace("</s>", "")
+            new_text = new_text.replace("<s>", "")
```

Comment:
Do these just signify the start and end of the stream?

Reply:
Not quite! I'm really not sure why they show up for a few models while for `TextGeneration` this isn't needed.

I saw something like `</s><s>Streaming Text</s>` But yea just removed so it looks clean